protocols/delegate-manager/test/DelegateManager.js

```diff
@@ -0,0 +1,141 @@
+const ERC20 = artifacts.require('FungibleToken')
+const Swap = artifacts.require('Swap')
+const Indexer = artifacts.require('Indexer')
+const Delegate = artifacts.require('Delegate')
+const DelegateFactory = artifacts.require('DelegateFactory')
+const DelegateManager = artifacts.require('DelegateManager')
+const { equal, passes } = require('@airswap/test-utils').assert
+const { padAddressToLocator } = require('@airswap/test-utils').padding
+
+contract('DelegateManager Integration Tests', async accounts => {
+  const STARTING_BALANCE = 700
+  const INTENT_AMOUNT = 250
+  let owner = accounts[0]
+  let notOwner = accounts[2]
+  let tradeWallet = accounts[1]
+  let DAI_TOKEN
+  let WETH_TOKEN
+  let stakeToken
```

Comment:
is there a reason DAI and WETH token variables are named in caps, and then stake token is in camel case?

Reply:
no particular reason. I will update.